sustainml_cpp/include/sustainml_cpp/orchestrator/OrchestratorNode.hpp

```diff
@@ -174,16 +173,26 @@ class OrchestratorNode
     /**
      * @brief Used to retrieve the associated OrchestratorNodeHandle.
      */
-    inline std::weak_ptr<OrchestratorNodeHandle> get_handler()
+    inline OrchestratorNodeHandle& get_handler()
     {
-        return handler_;
+        return *handler_;
     }
 
     /**
      * @brief Used to retrieve the associated OrchestratorNodeHandle.
      */
     void print_db();
 
+    /**
+     * @brief Called by the user to run the run.
+     */
+    void spin();
+
+    /**
+     * @brief Stops the execution of the node.
+     */
+    static void terminate();
+
```

Comment:
Add a new method `void destroy()`
This new method will:
```
void destroy()
{
  if(active_)
  {
    .... // what it is done currently in the destructor
     handler_ = nullptr;
     active_ = false;
  }
```

Add a new bool `active_` to the class and call `destroy()` from the destructor (this is in case we accidentally forget to call `destroy()` manually from outside somewhere else).